sql/schema.sql

```diff
@@ -889,6 +890,12 @@ BEGIN;
 END;
 
 
+--https://github.com/gratipay/gratipay.com/pull/4214
+BEGIN;
+  ALTER TABLE teams DROP COLUMN todo_url;
+END;
```

Comment:
@JessaWitzel Close! Rather than modifying `sql/schema.sql` directly, what we want is a file at `sql/branch.sql` (it shouldn't exist on your branch; you have to create it and add it to Git). If there's a file at `sql/branch.sql` then our deploy script [takes care of](https://github.com/gratipay/gratipay.com/blob/2fcc8a36d89aae6ac46f91dd7fa8bfcc038b4d7a/deploy.sh#L66-L85) appending it to `sql/schema.sql` after running `sql/branch.sql` against the production database. Make sense?